### DISCUSSION TITLE
🌴 Java基础入门 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#63] [Announcements]
# 🌴 Java基础入门 | HackWay技术学习路线

学编程最为重要的是实战，Java也不例外。

https://hackway.org/docs/devops/basic/Java%E5%9F%BA%E7%A1%80%E5%85%A5%E9%97%A8

**Comment by ghost** [comment]
这个视频可以换成2024动力节点的视频，B站讲的最好的一个。
